_includes/join-us/join-us-contribute-to-strimzi-band.html

```diff
@@ -0,0 +1,120 @@
+<div class="join-us-band">
+  <div class="component-slim grid-wrapper">
+    <div class="width-12-12">
+      <h2>Contribute to Strimzi</h2>
+
+      <p>Contributing to an open-source project is not just about writing code. 
+        There are many different ways you can contribute to Strimzi.</p>
+
+      <h3>Join the community</h3>
+
+      <p>You can start by joining the community. 
+        We have a <a href="https://slack.cncf.io/"><code>#strimzi</code> channel on the CNCF Slack workspace</a> which you can join and meet other Strimzi users. 
+        We also have <a href="https://lists.cncf.io/g/cncf-strimzi-users">user</a> and <a href="https://lists.cncf.io/g/cncf-strimzi-dev">developer</a> mailing lists. 
+        You can also come to our <a href="https://github.com/strimzi/strimzi-kafka-operator#strimzi-community-meetings">community calls</a> which happen every two weeks. 
+        On the community calls, you can learn about the current work or future plans. 
+        Don’t forget that in all these places you should follow the <a href="https://github.com/strimzi/governance/blob/main/CODE_OF_CONDUCT.md">code of conduct</a> and just be nice to others!</p>
+
+      <h3>Spread the word</h3>
+
+      <p>The easiest way you can help Strimzi is by spreading the word. 
+        It can start by something simple: <a href="https://github.com/strimzi">star Strimzi repositories on GitHub</a>, subscribe to <a gref="https://youtube.com/c/Strimzi">our YouTube channel</a> or <a href="https://twitter.com/strimziio">follow us on Twitter</a> and like / retweet our tweets. 
+        You can of course also talk about Strimzi on your social media accounts to help more people learn about our project. 
```

Comment:
maybe even adding something like "and don't forget to tag us!" ;-)

Reply:
Better now?

Reply:
Yep!